search/src/main/java/edu/artic/search/SearchBaseViewModel.kt

```diff
@@ -94,13 +93,20 @@ open class SearchBaseViewModel @Inject constructor(
                 navigateTo.onNext(Navigate.Forward(NavigationEndpoint.Web))
             }
             is SearchCircularCellViewModel -> {
-                viewModel.artWork?.let { articObject ->
-                    navigateTo.onNext(
-                            Navigate.Forward(
-                                    NavigationEndpoint.ArtworkOnMap(articObject)
-                            )
-                    )
+                viewModel.artWork?.let { arcticObject ->
```

Comment:
Please replace all instances of `arctic` with `artic`.

Reply:
Updated.